src/basic/user-util.h

```diff
@@ -80,7 +80,8 @@ int take_etc_passwd_lock(const char *root);
 #define UID_MAPPED_ROOT ((uid_t) (INT32_MAX-1))
 #define GID_MAPPED_ROOT ((gid_t) (INT32_MAX-1))
 
-#define ETC_PASSWD_LOCK_PATH "/etc/.pwd.lock"
+#define ETC_PASSWD_LOCK_NAME ".pwd.lock"
```

Comment:
NAME → FILENAME?

NAME is so generic

this would match how we have path_is_valid() and filename_is_valid()